src/app/components_ngrx/work-item-event-wrapper/work-item-event-wrapper.component.less

```diff
@@ -1,10 +1,10 @@
 @import (reference) '../../../assets/stylesheets/_base';
 
 .f8-event {
-  &-content {
+  &__list {
     li {
       text-align: center;
-      &.f8-toggle-caret {
+      &.f8-comment__toggle {
```

Comment:
This must nesting seems unnecessary when you have a very specific element level class already.